src/scripts.js

```diff
@@ -133,19 +135,32 @@ const loadRecipeCard = (event) => {
 };
 
 
-const loadSearchPage = (array) => {
-  searchPage.innerHTML = "";
-  loadPage(searchPage);
-  array.map(recipe => searchPage.innerHTML += `
+const loadPageResults = (array, page) => {
+  page.innerHTML = "";
+  loadPage(page);
+  array.forEach(recipe => page.innerHTML += `
     <article class="recipe-card recipe ${createKebab(recipe.name)} ">
       <img class="recipe-card-img" src="${recipe.image}">
       <p class="recipe-card-name">${recipe.name}</p>
       <p class="recipe-card-cost">${recipe.getIngredientsCost()}</p>
       <button class="recipe-card-button ${toggleFavoriteButton(recipe)}"></button>
     </article>
-  `);
+  `
+  );
 };
 
+const blurCard = () => {
+  currentUser.recipesToCook.forEach( recipe => {
+
+    if(!currentUser.hasSufficientIngredientsFor(recipe)){
+      const recipeCard = document.querySelector(`.${createKebab(recipe.name)}`)
+      // recipeCard.classList.add('blur');
+      recipeCard.insertAdjacentHTML('afterbegin', '<p class="more-ingredients">Not enough ingredients</p>')
```

Comment:
Luckily I happened to catch a small bug here - if you add "dirty steve's wing sauce" to your cook list the querySelector doesn't work because the apostrophe in Steve's screws up the the DOM element's class. We can modify your kebabCase function to exclude apostrophes - should fix the problem.